fe/lib/components/LocationSuggest/components/MapSelect.tsx

```diff
@@ -0,0 +1,176 @@
+        longitude: debouncedLatLong[1],
+      },
+      first: 3,
+      schemeId,
+    },
+    fetchPolicy: 'no-cache',
+  });
+
+  const nearestLocations =
+    data?.nearestLocations ?? previousData?.nearestLocations;
+
+  const isDesktopOrAbove = responsivevalue({
+    mobile: false,
+    desktop: true,
+  });
+
+  const nearestLocationsCards = () => (
+    <Box borderRadius="standard" background="surface">
+      <Stack dividers space="none">
+        <Box
+          paddingX="medium"
+          paddingY={{ desktop: 'xsmall', tablet: 'small', mobile: 'small' }}
+        >
+          <Columns space="small" alignY="center">
+            <Column>
+              <Text weight="strong" size="small">
+                {`Locations near ${formatPoint(latLong)}`}
+              </Text>
+            </Column>
+            {isDesktopOrAbove ? (
+              <Column width="content">
+                <ButtonIcon
+                  icon={<IconClear />}
+                  tone="secondary"
+                  id="clear-nearest-suggestions-button"
+                  label="Clear Suggestions"
+                  onClick={() => setShowSuggestions(false)}
+                />
+              </Column>
+            ) : null}
+          </Columns>
+        </Box>
+        {nearestLocations?.map((location) => (
+          <Box
+            className={styles.card}
+            cursor="pointer"
+            padding="medium"
+            key={location.id.value}
+            onClick={() => onLocationSelected(location)}
+          >
+            <Box position="relative">
+              <Stack space="small">
+                <Text size="small">{location.contextualName}</Text>
+                <InlineCode>{location.id.value}</InlineCode>
+              </Stack>
+            </Box>
+          </Box>
+        ))}
+      </Stack>
+    </Box>
+  );
+
+  const noLocationsNotice = () => (
+    <Alert
+      tone="info"
+      onClose={() => setShowSuggestions(false)}
+      closeLabel="Close"
+    >
+      <Text>{`No locations found.`}</Text>
+    </Alert>
+  );
+
+  const hasNearestLocations = nearestLocations && nearestLocations.length > 0;
+  const suggestionsOffset: [number, number] = [-40, 130];
+
+  return (
+    <Box>
+      <Map
+        center={center}
+        defaultCenter={initialLocation}
+        defaultZoom={DEFAULT_ZOOM}
+        height={isDesktopOrAbove ? 1500 : 600}
+        onClick={({ latLng }) => {
+          setLatLong(latLng);
+          setCenter(latLng);
+          setShowSuggestions(true);
+        }}
+        animate={true}
+        animateMaxScreens={100}
+      >
+        <Marker
+          color={color.foreground.formAccentLight}
+          anchor={latLong}
+          onClick={() => setCenter(latLong)}
+        />
+        {/* View for desktops and above */}
+        {showSuggestions && isDesktopOrAbove && (
+          <Overlay anchor={latLong} offset={suggestionsOffset}>
+            {hasNearestLocations
+              ? nearestLocationsCards()
```

Comment:
It's typically preferable to use React syntax when rendering a component 🙂  e.g.
```suggestion
              ? <NearestLocationCards />
```